app/controllers/companies_controller.rb

```diff
@@ -93,6 +94,45 @@ def destroy
   end
 
 
+  def get_dinkurs_events
```

Comment:
In anticipation of expanding "events" to include local-only events as well, this might be named `get_company_events`.